.rubocop.yml

```diff
@@ -178,10 +184,7 @@ Naming/UncommunicativeMethodParamName:
 Naming/VariableNumber:
   Enabled: false
 
-Performance/Casecmp:
```

Comment:
This cop no longer exists as far as I can tell.

Reply:
Agreed.